pkgs/os-specific/windows/mingw-w64/common.nix

```diff
@@ -1,16 +1,11 @@
 { fetchurl }:
 
 rec {
-  version = "5.0.3";
+  version = "5.0.4";
   name = "mingw-w64-${version}";
 
   src = fetchurl {
     url = "mirror://sourceforge/mingw-w64/mingw-w64-v${version}.tar.bz2";
-    sha256 = "1d4wrjfdlq5xqpv9zg6ssw4lm8jnv6522xf7d6zbjygmkswisq1a";
+    sha256 = "00zq3z1hbzd5yzmskskjg79xrzwsqx7ihyprfaxy4hb897vf29sm";
   };
-
-  configureFlags = [
-    "--enable-idl"
-    "--enable-secure-api"
-  ];
```

Comment:
I’m pretty sure we want to al least retain the secure-api. Otherwise this will result in frequent missing symbol issues.

Reply:
These weren't being used - only name & src are inherited in the derivation below. We can add them back though.